test/components/connect.spec.js

```diff
@@ -332,12 +332,20 @@ describe('React', () => {
       expect(decorated.state.data.testFunc).toBeA('function')
       expect(decorated.state.data.deferredFetch).toEqual(undefined)
 
-      decorated.state.data.testFunc('A', 'B')
+      decorated.state.data.testFunc('A', 'B').then(value => expect(value).toEqual({ T: 't' }))
```

Comment:
Ah, so the way I did this is the promise resolves not with the PromiseState but with just the value (it was less code this way, and I didn't see any use for the rest of the PromiseState, but now I'm thinking maybe `meta` would have been useful)